flake.lock

```diff
@@ -486,11 +486,11 @@
     },
     "nixpkgs-unstable": {
       "locked": {
-        "lastModified": 1724479785,
-        "narHash": "sha256-pP3Azj5d6M5nmG68Fu4JqZmdGt4S4vqI5f8te+E/FTw=",
+        "lastModified": 1725103162,
+        "narHash": "sha256-Ym04C5+qovuQDYL/rKWSR+WESseQBbNAe5DsXNx5trY=",
         "owner": "NixOS",
         "repo": "nixpkgs",
-        "rev": "d0e1602ddde669d5beb01aec49d71a51937ed7be",
+        "rev": "12228ff1752d7b7624a54e9c1af4b222b3c1073b",
```

Comment:
we're sure we can safely update nixpkgs-unstable?

Reply:
that's the guidance I got from @cor - to update it and if there are any issues to raise and try to fix. This is what I did last few times (last update 5 days ago https://github.com/unionlabs/union/commit/7e38400a540f7f1c5d11f628cd9caa7bf4d0fca3)